File: template/template.kt
// My Solution : https://github.com/ywcheong/solved-baekjoon

// Header
// FILL HERE: Import External Library

private var TESTCASE_ID = 1
private fun <T> equal(left: T, right: T) {
    if (left == right) {
        println("Testcase $TESTCASE_ID: OK ($left == $right)")
    } else {
        println("Testcase $TESTCASE_ID: FAIL ($left != $right)")
    }
    TESTCASE_ID++
}

// Implementation
// FILL HERE: Implement


// Testing
private fun test() {
    println("WARNING: TEST MODE")

    // FILL HERE: Write Testcases

    println("TEST DONE")
}

// Submit
private fun submit() {
    // FILL HERE: Get Inputs
}

// Case-switch
fun main() {
    test()
    submit()
}

File: template/template.py
# My Solution : https://github.com/ywcheong/solved-baekjoon

import sys


def solve(x):
    """write your solution here"""
    return x


def test():
    def check(left, right):
        if left == right:
            print(f"Test pass")
        else:
            print(f"Test fail: {left} != {right}")

    """write your testing here"""
    check(solve(1), 1)
    check(solve(2), 2)
    check(solve(3), 3)


def main():
    def input_one(given_type):
        return given_type(sys.stdin.readline().strip())

    def input_list(given_type):
        return list(map(given_type, input_one(str).split()))

    """write your i/o here"""
    x = input_one(int)
    print(solve(x))


test()
main()
